src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -475,177 +473,177 @@ def build_process_costs(
                     initialize=value(self.total_BEC),
                     bounds=(0, 1e4),
                     doc="Electrical, materials and labor ancillary cost in $MM",
-                    units=getattr(pyunits, "MUSD_" + CE_index_year),
+                    units=CE_index_units,
                 )
 
                 self.instrumentation_costs = Var(
                     initialize=value(self.total_BEC),
                     bounds=(0, 1e4),
                     doc="Ancillary cost in $MM",
-                    units=getattr(pyunits, "MUSD_" + CE_index_year),
+                    units=CE_index_units,
                 )
 
                 self.plant_services_costs = Var(
                     initialize=value(self.total_BEC),
                     bounds=(0, 1e4),
                     doc="Ancillary cost in $MM",
-                    units=getattr(pyunits, "MUSD_" + CE_index_year),
+                    units=CE_index_units,
                 )
 
                 # buildings cost variables
                 self.buildings_costs = Var(
                     initialize=value(self.total_BEC),
                     bounds=(0, 1e4),
                     doc="Buildings cost in $MM",
-                    units=getattr(pyunits, "MUSD_" + CE_index_year),
+                    units=CE_index_units,
                 )
 
                 self.process_buildings_costs = Var(
                     initialize=value(self.total_BEC),
                     bounds=(0, 1e4),
                     doc="Process buildings cost in $MM",
-                    units=getattr(pyunits, "MUSD_" + CE_index_year),
+                    units=CE_index_units,
                 )
 
                 self.auxiliary_buildings_costs = Var(
                     initialize=value(self.total_BEC),
                     bounds=(0, 1e4),
                     doc="Auxiliary buildings cost in $MM",
-                    units=getattr(pyunits, "MUSD_" + CE_index_year),
+                    units=CE_index_units,
                 )
 
                 self.site_improvements_costs = Var(
                     initialize=value(self.total_BEC),
                     bounds=(0, 1e4),
                     doc="Site improvements buildings cost in $MM",
-                    units=getattr(pyunits, "MUSD_" + CE_index_year),
+                    units=CE_index_units,
                 )
 
                 # engineering, procurement and construction management cost variables
                 self.epcm_costs = Var(
                     initialize=value(self.total_BEC),
                     bounds=(0, 1e4),
                     doc="EPCM cost in $MM",
-                    units=getattr(pyunits, "MUSD_" + CE_index_year),
+                    units=CE_index_units,
                 )
 
                 self.equipment_installation_costs = Var(
                     initialize=value(self.total_BEC),
                     bounds=(0, 1e4),
                     doc="Equipment installation EPCM cost in $MM",
```

Comment:
Given the change below, I do not think we should keep the "$MM" in the doc string, as it can quite easily be wrong. For determining the units of something, it is far better to directly query the units attribute than to look at a doc string that is likely static.

Reply:
I took out all of the docstring and report references to dollars, $, or USD. The method to autodetect reference dictionary units still looks for "$", that's something that will need to be generalized when we do that currency update.

Reply:
Also, the report method has a new line to report the currency units.